main.go

```diff
@@ -209,19 +153,23 @@ func mainWithExitCode() exitCode {
 		return failure("Failed to parse the included tracers: %v", err)
 	}
 
-	log.Infof("Assigned ProjectID: %d HostID: %d", args.projectID, environment.HostID())
-
-	metadataCollector := hostmetadata.NewCollector(args.collAgentAddr, environment)
+	metadataCollector := hostmetadata.NewCollector(args.collAgentAddr)
+	metadataCollector.AddCustomData("os.type", "linux")
 
-	// TODO: Maybe abort execution if (some) metadata can not be collected
-	hostMetadataMap = metadataCollector.GetHostMetadata()
-
-	if bpfJITEnabled, found := hostMetadataMap["host.sysctl.net.core.bpf_jit_enable"]; found {
-		if bpfJITEnabled == "0" {
-			log.Warnf("The BPF JIT is disabled (net.core.bpf_jit_enable = 0). " +
-				"Enable it to reduce CPU overhead.")
-		}
+	hostname, _ := os.Hostname()
+	metadataCollector.AddCustomData("host.name", hostname)
```

Comment:
There's a semantic change here in that we're no longer entering the root namespace to retrieve hostname.

Reply:
Added functionality back to fetch this information from root namespace with https://github.com/elastic/otel-profiling-agent/pull/90/commits/21eeedc4c471da0f56c095d7ef46ce4bbbc7301c.